src/plugins/mce/plugin.c

```diff
@@ -38,10 +38,30 @@ typedef struct _MceData
     NRequest       *request;
     NSinkInterface *iface;
     gchar          *pattern;
+    gboolean       playing;
```

Comment:
Nitpick: I think the variable name indentation should match the other line, * is not part of the name.